helm-chart/values.yaml

```diff
@@ -49,7 +49,9 @@ controlPlane:
       cpu: "1"
   # Security context for the pod or container
   securityContext: {}
-  #extra_content: {}  # Extra content (currently commented out)
+  #extraContent: # Extra content, needed if you want to configure a reverse proxy for control plane.
```

Comment:
question: why in extraContent ?

Reply:
To dynamically inject new configuration without needing to update the control plane configuration on Helm every time, it minimizes the need for frequent changes. WDYT?

Reply:
But don't we lost helm generated documentation?
I see no difference with adding the field for real

Reply:
~~reverse~~ => forward

Reply:
Updated and fixed.